src/App.js

```diff
@@ -109,10 +109,10 @@ const App = (props) => {
                     }
                 }}>
                 <NotificationsPortal store={ store }/>
-                <Routes childProps={ props } />
```

Comment:
@AsToNlele Do you know what these `childProps` were for?

Reply:
Inside were router props like `history`, `location` etc which are not needed